cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -427,28 +427,62 @@ update_host_ld_cache() {
   ldconfig -r "${ROOT_MOUNT_DIR}"
 }
 
+usage() {
+  echo "usage: $0 [-p|-s]"
+  echo "Default behavior installs all components needed for the Nvidia driver."
+  echo "  -p: Install cross toolchain package only"
+  echo "  -s: Skip cross toolchain package installation"
+}
+
 main() {
-  lock
-  load_etc_os_release
-  configure_kernel_module_locking
-  if check_cached_version; then
-    configure_cached_installation
-    verify_nvidia_installation
-    info "Found cached version, NOT building the drivers."
-  else
-    info "Did not find cached version, building the drivers..."
-    download_nvidia_installer
-    download_kernel_src
-    set_compilation_env
+  while getopts ":psh" opt; do
```

Comment:
Please move these to a separate function parse_opt.

Reply:
Thanks. Updated.